test/index.spec.js

```diff
@@ -0,0 +1,11 @@
+const heml = require('../packages/heml')
+const fixtures = require('./__fixtures__')
+
+describe('snapshots', () => {
```

Comment:
I was thinking we could drop this describe and rename the file "snapshots.spec.js" to streamline the output a bit